pkg/util/steps/condition.go

```diff
@@ -116,7 +117,7 @@ func enrichConditionTimeoutError(f conditionFunction, originalErr error) error {
 	return api.NewCloudError(
 		http.StatusInternalServerError,
 		api.CloudErrorCodeDeploymentFailed,
-		"", message+"Please retry, if issue persists: raise azure support ticket",
+		"", message+"Please retry, if issue persists: raise Azure support ticket",
```

Comment:
nit: while we're capitalizing this, can we get a space between `message` and `Please retry ...`? Or to make it even more grammatically correct, `<message>, please retry. If the issue persists raise an Azure support ticket.`

Reply:
Good point, I should done this in the first place. I did a slight variation on what you suggested to avoid removing the period at the end of each error message, so let me know what you think.